components/Review/installation.rst

```diff
@@ -0,0 +1,18 @@
+Installation
+============
+
+We assume you're familiar with `Composer <http://packagist.org>`_, a dependency manager for PHP.
+Use following command to add the component to your `composer.json` and download package.
+
+If you have `Composer installed globally <http://getcomposer.org/doc/00-intro.md#globally>`_.
+
+.. code-block:: bash
+
+    $ composer require sylius/review:*
```

Comment:
`sylius/review` without `:*` will result in Composer getting the best version possible (supports semver) :)